configuration/device-tree.md

```diff
@@ -405,56 +479,58 @@ Here are some examples of different types of properties, with parameters to modi
 		u32_1 =       <&test>,"u32s:4";
 		u64_0 =       <&test>,"u64s#0";
 		u64_1 =       <&test>,"u64s#8";
-		bool1 =       <&test>,"bool1?";
+		bool1 =       <&test>,"bool1!";
 		bool2 =       <&test>,"bool2?";
-		only1 =       <0>,"+1-2";
-		only2 =       <0>,"-1+2";
-		toggle1 =     <0>,"=1";
+		entofr =      <&test>,"english",
+		              <&test>,"french{hello=bonjour,goodbye='au revoir',weekend}";
+		pi_mac =      <&test>,"mac[{1=b8273bfedcba,2=b8273b987654}";
+		spibus =      <&test>,"spi:0[0=",<&spi0>,"1=",<&spi1>,"2=",<&spi2>;
+
 		toggle2 =     <0>,"=2";
 		not1 =        <0>,"!1";
-		not2 =        <0>,"!2";
+
+		only1 =       <0>,"+1-2";
+		only2 =       <0>,"-1+2";
+		enable1 =     <0>,"=1";
+		disable2 =    <0>,"!2";
```

Comment:
Is the `toggle2` example doing something different to the `enable1` example?
(and I guess similarly `not1` and `disable2`)

Reply:
They should have been deleted.